internal/builders/go/README.md

```diff
@@ -145,6 +145,33 @@ The configuration file accepts many of the common fields GoReleaser uses, as you
 
 If you think you need support for other variables, please [open an issue](https://github.com/slsa-framework/slsa-github-generator/issues/new).
 
+### Multi-platform builds
+
+Similar to the goreleaser workflow, it's easy to generate binaries for multiple platforms. To accomplish this, we can use the [maxtrix functionality](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) of github actions and several config files. In the below code sample, you'll see that we have a `strategy` section which lists the platforms and architectures to build for. These reference a `config-file` property at the bottom which will select the correct config for that platform.
```

Comment:
NOTE: goreleaser uses a single config file, so it's more user-friendly. Let's just remove the the `Similar to the goreleaser workflow`, and starts with `It's easy`.